reo/src/data_manager.py

```diff
@@ -406,16 +487,16 @@ def _get_REopt_cost_curve(self, techs):
                     if u == 0 or u_cap == 0:
                         switch_rebate = True
 
-                    # start at second point, first is always zero
+                    # TODO not true: start at second point, first is always zero
```

Comment:
"not true" because of affine cost curves? Do we need to address this now?

Reply:
I don't remember the fix I did for GHP with "first is always zero", but I worked around the existing framework. So I'm deleting the TODO